src/docs/testing/integration-tests/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Integration Testing
+description: Learn how to write integration tests
+---
+
+This page describes how to use the [integration_test][] package to run
+integration tests. Tests written using this package have the following
+properties:
+ 
+- Compatibility with the `flutter drive` command, for running tests on a
+physical device or emulator.
+- The ability to be run on [Firebase Test Lab], enabling automated testing on a
+variety of devices.
+- Compatibility with [flutter_test] APIs, enabling tests to be written in a
+similar style as [widget tests][]
+
+## Overview
+
+If you're developing a Flutter app, you are probably writing the code on a
+desktop computer, called the **host** machine, and running the app on a mobile
+device or browser, called the **target**.
+
+There are three types of tests that Flutter supports. A **unit** test verifies
+the behavior of a method or class. A **widget test** verifies the behavior of
+Flutter widgets without running the app itself. An **integration test** runs the
+full app on the target device and reports the results back to the host machine.
+
+The flutter_driver package runs integration tests written in Dart on a target
+device and reports the results to the host.
+
+The [integration_test][] package lets you run tests directly on the target
+device. This allows you to run integration tests on multiple Android or iOS
+devices using Firebase TestLab. These tests can also be run using
```

Comment:
Before you used "Firebase Test Lab". I googled and it looks like it is officially three words, not two.